packages/lodestar/src/api/rest/routes/beacon/syncing.ts

```diff
@@ -0,0 +1,20 @@
+import {IFastifyServer} from "../../index";
+import * as fastify from "fastify";
+import {IApiModules} from "../../../interface";
+
+export const registerSyncingeEndpoint = (fastify: IFastifyServer, modules: IApiModules): void => {
+  fastify.get<fastify.DefaultQuery, {}, unknown>('/syncing', {}, async (request, reply) => {
```

Comment:
probably `/node/syncing`? Same for other end points.

Reply:
https://github.com/ChainSafe/lodestar/blob/920161e12029de8aa786cd82b324d90b63d5ca8d/packages/lodestar/src/api/rest/index.ts#L50